### ISSUE TITLE
Clean up mode selection in map options menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#157]
```
The current mode selection interface in the map options menu is confusing:
- Selection mode
- Panning mode
- Zooming mode
- Fit-to-window mode
should be reorganized as drop down options from a Mode item. This needs to be 
coordinated with the revised zooming and panning operation (see also #22, #114,
and #147)
```

Original issue reported on code.google.com by `lanse...@gmail.com` on 26 Jun 2015 at 1:48

**Comment by jkoschinsky** [comment]
Verified in GeoDa 1.7.63 Mac OSX
Verify in Windows

**Comment by jkoschinsky** [comment]
Replaced with map view icons.
Verified fix in GeoDa 1.7.89 (Win 7 Professional (64-bit)).
